README.md

```diff
@@ -154,6 +152,33 @@ YACHT can be run via the command line `yacht <module>`. Now it has three four ma
   | k                 | the length of k-mer |
   | outfolder         | the path to a folder where the downloaded file is expected to locate |
 
+- The `sketch` module has two submodules: `ref` and `sample`:
+  
+  + `ref` is used to sketch fasta files and make them as a reference database
```

Comment:
We'll want to make clear that the output `ref.sig.zip` will be a container of multiple sketches, one for each of the FASTA/Q files in the given folder, and if a single FASTA/Q file, then one sketch per record.

Reply:
We have already provided an illustration [here](https://github.com/KoslickiLab/YACHT/blob/474208ff6845d9a634737e180cb4681773af0397/README.md?plain=1#L221-L233).